lib/features/local_live_log.js

```diff
@@ -28,6 +29,7 @@ export default class TaskclusterLogs {
     Docker container used in the linking process.
     */
     this.container = null;
+    this.token = crypto.randomBytes(64).toString('hex');
```

Comment:
So I thought this was a potentially bad use of entropy... Fearing that this could block for a long time, if we were very unlucky. Honestly, I would have used `slugid.v4()`, but I agree that this is not a good idea.

Anyways, looking at `uuid` underlying `slugid` I realize that it uses `crypto.randomBytes()` too...

So I dugg down to see if it was safe to pull a lot of data out of this... and it seems that it is: http://stackoverflow.com/a/18131768/68333
Basically it's the same as pseudo random, except it'll throw an error if there isn't enough entropy, but once entropy is acquired it'll never complain.

So I'm happy with this. Just wanted to add the background information here.

---
Also considering that a slugid is just as safe, why don't we just use that... It's sorter an already URL-safe base64 encoded. 16 or 64 bytes probably doesn't matter, 64 is a bit overkill; it's like we're not giving the NSA any chance at guessing right :)